.github/.cache-version

```diff
@@ -1,3 +1,3 @@
 # Update this file to invalidate **all** cache on GitHub actions (use sparingly)
 
-version: v2
+version: v3
```

Comment:
❓what prompted this?

Reply:
I'm trying to gauge why the linting process doesn't seem to want to find xstate